addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessage.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.List;
+
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.openhab.binding.rfxcom.internal.config.RFXComBridgeConfiguration;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+
+/**
+ * RFXCOM data class for control message.
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComInterfaceControlMessage extends RFXComBaseMessage {
+
+    private byte[] data = new byte[14];
+
+
+    public RFXComInterfaceControlMessage(RFXComInterfaceMessage.TransceiverType transceiverType, RFXComBridgeConfiguration configuration) {
+        data[0] = 0x0D;
+        data[1] = RFXComBaseMessage.PacketType.INTERFACE_CONTROL.toByte();
+        data[2] = 0;
+        data[3] = 2;
+        data[4] = RFXComInterfaceMessage.Commands.SET_MODE.toByte();
+        data[5] = transceiverType.toByte();
+        data[6] = 0;
```

Comment:
In RF it is *never* right to blindly default to max power. That simply increases the noise floor over the widest possible area for no reason. Let's put a stake in the ground. I've added xmit power as a config option so it can be increased if necessary. Most hex mode strings floating around the net seem to set 0 anyway. I can't remember ever seeing one with a non-zero value, never mind anyone actually mentioning that setting xmit power was an option.